pkg/commands/git_commands/file_loader.go

```diff
@@ -100,6 +155,16 @@ type FileStatus struct {
 	PreviousName string
 }
 
+func (fileLoader *FileLoader) gitDiffNumStat() (string, error) {
```

Comment:
Three things:
1) Just confirming that this can't be bundled into the `git status` command?
2) How expensive is this command to run? If it's expensive, we may need to run it asynchronously
3) I've noticed that this doesn't include untracked files. Is there a way to include those?